cf_units/__init__.py

```diff
@@ -1883,27 +1887,26 @@ def convert(self, value, other, ctype=FLOAT64):
             32.0
             >>> a64 = np.arange(10, dtype=np.float64)
             >>> c.convert(a64, f)
-            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6, \
- 46.4,  48.2])
+            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6,
+                    46.4,  48.2])
```

Comment:
They aren't running using ``python setup.py test``. I did a ``py.test --doctest-modules`` and got some failures including this one. The line break drove me crazy, so I reduced the test arrays to size 3 to fit on a single line. There are three other failures, but I don't think I should fix those (they look minor).